hack/kubernetes/allinone.sh

```diff
@@ -17,17 +17,31 @@ install_pouch_ubuntu() {
 	sudo add-apt-repository "deb http://mirrors.aliyun.com/opsx/pouch/linux/debian/ pouch stable"
 	sudo apt-get update
 	sudo apt-get install pouch
-	sudo service pouch start
+	sudo systemctl enable pouch
+	sudo systemctl start pouch
 }
 
 install_pouch_centos() {
 	sudo yum install -y yum-utils
 	sudo yum-config-manager --add-repo http://mirrors.aliyun.com/opsx/opsx-centos7.repo
 	sudo yum update
 	sudo yum install pouch
```

Comment:
I think here should add "-y" parameter, or you will be asked to input `y` or `n`

Reply:
Right. :)